04/passenger-service/src/test/java/cz/muni/fi/airportmanager/passengerservice/service/PassengerServiceTest.java

```diff
@@ -0,0 +1,185 @@
+package cz.muni.fi.airportmanager.passengerservice.service;
+
+import cz.muni.fi.airportmanager.passengerservice.entity.Notification;
+import cz.muni.fi.airportmanager.passengerservice.entity.Passenger;
+import cz.muni.fi.airportmanager.passengerservice.model.CreatePassengerDto;
+import cz.muni.fi.airportmanager.passengerservice.repository.PassengerRepository;
+import io.quarkus.test.InjectMock;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.vertx.RunOnVertxContext;
+import io.quarkus.test.vertx.UniAsserter;
+import io.smallrye.mutiny.Uni;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@QuarkusTest
+class PassengerServiceTest {
+
+    @InjectMock
+    PassengerRepository passengerRepository;
+
+    @Inject
+    PassengerService passengerService;
+
+    private Passenger createTestPassenger() {
```

Comment:
private method to the end of class please

Reply:
moved